### PR TITLE
Release/0.0.1.dev7

### PR DESCRIPTION
**Pull request by descent3me**
**What this PR does / why we need it**:

Prepare v0.0.1.dev7 release

**Which issue(s) this PR fixes**

Fixes #

**Special notes for your reviewer**:
@spotify/alf 

**Release note**:

```release-note

```

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,10 +1,22 @@
 Changelog
 =========
 
+0.0.1.dev7 (2018-06-15)
+-----------------------
+Changed
+~~~~~~~
+* Update gordon-dns to 0.0.1.dev3
```
Comment (econchick) [inline CHANGELOG.rst]: Periods? 😇

Comment (descent3me) [reply CHANGELOG.rst]: damnit! will fix!

**Comment by econchick** [comment]
Looks like there's some docs that need to be updated referring to the old interface names

**Comment by descent3me** [comment]
Yeah, I don't know why that breaks on this PR instead of the PR where I update the dependency.

**Pull request merged**